src/main/groovy/io/github/cosmicsilence/scalafix/ScalafixPlugin.groovy

```diff
@@ -60,23 +60,27 @@ class ScalafixPlugin implements Plugin<Project> {
     }
 
     private void configureTaskForSourceSet(SourceSet sourceSet,
-                                           ScalafixMainMode mode,
+                                           ScalafixMainMode taskMode,
                                            Task mainTask,
                                            Project project,
                                            ScalafixExtension extension) {
         def name = mainTask.name + sourceSet.name.capitalize()
-        def task = project.tasks.create(name, ScalafixTask, mode)
-        task.description = "${mainTask.description} in '${sourceSet.getName()}'"
-        task.group = mainTask.group
-        task.source = sourceSet.allScala.matching {
-            include(extension.includes.get())
-            exclude(extension.excludes.get())
+        def task = project.tasks.create(name, ScalafixTask) {
+            description = "${mainTask.description} in '${sourceSet.getName()}'"
+            group = mainTask.group
+            source = sourceSet.allScala.matching {
+                // This is applied after evaluating the project
```

Comment:
sug.: I think we can drop this comment as the whole method is now invoked inside `afterEvaluate`